src/uct/api/v2/uct_v2.h

```diff
@@ -772,6 +778,11 @@ typedef struct {
      * Memory Domains belong to the same device.
      */
     char              global_id[UCT_MD_GLOBAL_ID_MAX];
+
+    /**
+     * Bitmap of memory types that are suitable for non-blocking registration
+     */
+    uint64_t          reg_nonblock_mem_types;
```

Comment:
maybe add this field, and its corresponding bit, after the field "reg_mem_types" ?

Reply:
this would break combatibility

Reply:
v2 does not need to be compatible